packages/siwe-parser/lib/t-ipv4-ipv6.test.ts

```diff
@@ -0,0 +1,145 @@
+import { ParsedMessage } from "./abnf";
+const doUri = function doUri(uri: string) {
+  let msg14 = "";
+  msg14 += "service.org wants you to sign in with your Ethereum account:\n";
+  msg14 += "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2\n";
+  msg14 += "\n";
+  msg14 += "\n";
+  msg14 += `URI: ${uri}\n`;
+  msg14 += "Version: 1\n";
+  msg14 += "Chain ID: 1\n";
+  msg14 += "Nonce: 32891757\n";
+  msg14 += "Issued At: 2021-09-30T16:25:24.000Z";
+  return new ParsedMessage(msg14);
+};
+let result;
+
+describe("test IPv4 addresses", () => {
```

Comment:
If you could add those tests to the JSON tests (which can then be used by other libraries in other languages) that would be great

Reply:
Uumm. Not so sure about this one. That looks like a ton of work creating JSON objects with all of that test data in it. But I can see out it might be helpful to your overall project beyond just this `ParsedMessage` object. We can discuss this further.

Reply:
Yes, I will try to put all of the new tests into JSON objects so that they can be run similarly to `parsers.test.ts` or maybe fully integrated into that file. But I'd like to close this conversation for now and present the modified unit tests as a separate PR later.